airflow_clickhouse_plugin/sensors/clickhouse_sql_sensor.py

```diff
@@ -1,23 +1,56 @@
+from airflow.exceptions import AirflowException
 from airflow.sensors.sql_sensor import SqlSensor
 from airflow_clickhouse_plugin.hooks import clickhouse_hook
 
 
 class ClickHouseSqlSensor(SqlSensor):
     def __init__(
         self,
-        clickhouse_conn_id: str = "clickhouse_default",
-        database=None,
+        clickhouse_conn_id: str = 'clickhouse_default',
+        database: str = None,
         *args,
-        **kwargs
+        **kwargs,
```

Comment:
Unfortunately, usage `*args`/`**kwargs` seems to be unavoidable in case of Airflow since it is propagate through the whole initializers hierarchy. So it is ok to keep it for now.

But please list all of the `SqlSensor.__init__` named arguments at least. It will make `ClickHouseSqlSensor` much easier to use.